application-gateway-logviewer-goaccess/azuredeploy.json

```diff
@@ -0,0 +1,183 @@
+            "properties": {
+              "privateIPAllocationMethod": "Dynamic",
+              "publicIPAddress": {
+                "id": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]"
+              },
+              "subnet": {
+                "id": "[variables('subnetRef')]"
+              }
+            }
+          }
+        ]
+      }
+    },
+    {
+      "apiVersion": "2017-03-30",
+      "type": "Microsoft.Compute/virtualMachines",
+      "name": "[variables('vmName')]",
+      "location": "[parameters('location')]",
+      "dependsOn": [        
+        "[concat('Microsoft.Network/networkInterfaces/', variables('nicName'))]"
+      ],
+      "properties": {
+        "hardwareProfile": {
+          "vmSize": "[variables('vmSize')]"
+        },
+        "osProfile": {
+          "computerName": "[variables('vmName')]",
+          "adminUsername": "[parameters('adminUsername')]",
+          "adminPassword": "[parameters('adminPassword')]"
+        },
+        "storageProfile": {
+          "imageReference": {
+            "publisher": "[variables('imagePublisher')]",
+            "offer": "[variables('imageOffer')]",
+            "sku": "18.04-LTS",
+            "version": "latest"
+          },
+          "osDisk": {
+            "name": "[concat(variables('vmName'),'_osdisk')]", 
+            "caching": "ReadWrite",
+            "createOption": "FromImage"
+          }
+        },
+        "networkProfile": {
+          "networkInterfaces": [
+            {
+              "id": "[resourceId('Microsoft.Network/networkInterfaces',variables('nicName'))]"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Compute/virtualMachines/extensions",
+      "name": "[concat(variables('vmName'),'/newuserscript')]",
+      "apiVersion": "2015-05-01-preview",
+      "location": "[parameters('location')]",
+      "dependsOn": [
+        "[concat('Microsoft.Compute/virtualMachines/', variables('vmName'))]"
+      ],
+      "properties": {
+        "publisher": "Microsoft.Azure.Extensions",
+        "type": "CustomScript",
+        "typeHandlerVersion": "2.0",
+        "autoUpgradeMinorVersion": true,
+        "settings": {
+          "fileUris": [            
+            "https://appgwloganalyzergoaccess.blob.core.windows.net/loganalyzerservice/scripts/setup_vm.sh?st=2018-09-12T07%3A09%3A54Z&se=2050-09-13T07%3A09%3A00Z&sp=rl&sv=2018-03-28&sr=b&sig=y57URB%2FiGrAawyHhDb6waznnyHvSrOsAHEcF4cu9BFw%3D",
+            "https://appgwloganalyzergoaccess.blob.core.windows.net/loganalyzerservice/build/AppGatewayLogProcessor.zip?st=2018-09-12T06%3A56%3A42Z&se=2050-09-13T06%3A56%3A00Z&sp=rl&sv=2018-03-28&sr=b&sig=c9x6svGOKJOHvHEwCDzM9BSrO5FwPRMIDhuQ4m1OeWQ%3D"
+          ],
+          "commandToExecute": "[concat('sh setup_vm.sh ' , variables('appGwAccessLogsBlobSasUriVar'), ' ', variables('filterRegexForAppGwAccessLogsVar'))]"
+        }
+      }
+    }
+  ]
+}
```

Comment:
output the endpoint where user's can see the report (use reference() to get the fqdn)